code/GF_LOGGA_VIT.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="217.3px" height="111.8px" viewBox="0 0 217.3 111.8" enable-background="new 0 0 217.3 111.8" xml:space="preserve">
+<g>
```

Comment:
Seems like you're inside the GF_LOGGA_VIT.svg file, which is an image-file made by Guldfabriken (the company I made the site for), so that code is something that's out of my hands. The only files I've edited is the index.html, style.css, script.js and the readme.md file!